.github/workflows/ci.yml

```diff
@@ -44,9 +44,41 @@ jobs:
       run: make build
     - name: Check clean repository
       run: make check-clean-work-tree
-
-  test-race:
+  lint-windows:
     runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.DEFAULT_GO_VERSION }}
+          check-latest: true
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+      - name: Setup Environment
+        run: |
+          echo "GOPATH=$(go env GOPATH)" >> $GITHUB_ENV
+          echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
```

Comment:
This step is no longer needed anywhere.

See https://github.com/open-telemetry/opentelemetry-go/pull/4617 for reference.